CHANGELOG.md

```diff
@@ -1,3 +1,133 @@
+# v1.1.0 (2021-05-07)
```

Comment:
Note: this PR assumes the following PRs will be merged:
* https://github.com/bottlerocket-os/bottlerocket/pull/1541
* https://github.com/bottlerocket-os/bottlerocket/pull/1549
* https://github.com/bottlerocket-os/bottlerocket/pull/1545
* https://github.com/bottlerocket-os/bottlerocket/pull/1547

It also assumes we'll create and merge the PR to update aws-ecs-1 to kernel 5.10.

Reply:
https://github.com/bottlerocket-os/bottlerocket/pull/1555 is the ECS kernel bump PR.